### PR TITLE
🌱 viewer: display scorecard version, API results URL

### LINKED ISSUE #441: Viewer should display which version of Scorecard calculated the score
If a project has installed the Scorecard Action, the Action's result takes precedence over what's calculated by the cronjob. However, if the Action isn't up-to-date, it may be running an old version of Scorecard, which can impact the project's score.

Therefore, the viewer should display the version of Scorecard used to calculate that score, something like this:

Screenshot 2023-07-19 at 17 18 59

(Maybe add an alert if it isn't running the most recent version, so readers are aware?)

### PR DESCRIPTION
**Pull request by raghavkaul**
Update scorecard-viewer display. 

Also update URL from [securityscorecards.dev](securityscorecards.dev) -> [scorecard.dev](scorecard.dev).

Fixes #441.

### CODE REVIEW

**Comment by raghavkaul** [comment]
The new display looks like [this](https://deploy-preview-597--ossf-scorecard.netlify.app/viewer/?uri=github.com/ossf/scorecard):
Screenshot 2024-03-25 at 4 51 12 PM

**Comment by spencerschrock** [comment]
@raghavkaul can you update the PR description to close the associated issue?
https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue

File: scorecards-site/static/viewer/index.html [hunk 1]
```diff
@@ -850,6 +850,21 @@
...
               </div>
+              <div class="content"><small class="fw-semibold">API URL:</small>
+                {{#if data.repo.name}}
+                  {{#if data.repo.commit}}
+                    <a class="btn-link" href="https://api.scorecard.dev/{{data.repo.name}}?commit={{data.repo.commit}}" target="_blank" rel="noopener noreferrer">
+                      https://api.scorecard.dev/{{data.repo.name}}
+                    </a>
+                  {{else}}
+                    <a class="btn-link" href="https://api.scorecard.dev/{{data.repo.name}}" target="_blank" rel="noopener noreferrer">
+                      https://api.scorecard.dev/{{data.repo.name}}
```
Comment (spencerschrock) [inline scorecards-site/static/viewer/index.html:861]: these api links don't work, click through and you'll get 404. you're missing `projects` i think

Comment (raghavkaul) [reply scorecards-site/static/viewer/index.html]: Fixed in [dd058ac](https://github.com/ossf/scorecard-webapp/pull/597/commits/dd058acae1e503354467a78e2e48fb8dbd26b2c9).

File: scorecards-site/static/viewer/index.html [hunk 2]
```diff
@@ -850,6 +850,21 @@
...
+                    </a>
```
[the remark this answers is not in the document]
Comment (raghavkaul) [reply scorecards-site/static/viewer/index.html]: Fixed in [dd058ac](https://github.com/ossf/scorecard-webapp/pull/597/commits/dd058acae1e503354467a78e2e48fb8dbd26b2c9)

File: scorecards-site/static/viewer/index.html [hunk 3]
```diff
@@ -850,6 +860,15 @@
                   </a>
                   </div>
               </div>
+              <div class="content"><small class="fw-semibold">API URL:</small>
+                {{#if apiURL}}
+                  <a class="btn-muted-link" href="{{apiURL}}" target="_blank" rel="noopener noreferrer">
+                    https://api.scorecard.dev/{{data.repo.name}}
```
Comment (spencerschrock) [inline scorecards-site/static/viewer/index.html:866]: This link (even though the `href` works) should be:
```
https://api.scorecard.dev/projects/{{data.repo.name}}
```

Comment (spencerschrock) [reply scorecards-site/static/viewer/index.html:866]: Note: this doesn't have the commit part either, but I think thats ok

**Review by spencerschrock — APPROVED** [review]

File: scorecards-site/static/viewer/index.html (hunk 3 above)
Comment (raghavkaul) [reply scorecards-site/static/viewer/index.html]: added `/projects/` but leaving the commit out for readability purposes.

**Pull request merged**